src/Services/XMLTransform.php

```diff
@@ -94,6 +100,7 @@ public function transformFits($input_xml) {
       ];
     }
     else {
+      $fits_metadata['Droid'] = ['PUID' => $puid];
```

Comment:
If the $puid is wrapped in a conditional (line 88), should this also be wrapped?

Reply:
Makes sense. Done!